### PR TITLE
add scikit-time recipe

### PR DESCRIPTION
**Pull request by marscher**
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by marscher** [comment]
@clonker fyi

**Comment by marscher** [comment]
@conda-forge/help-python This PR is ready to be reviewed. Thanks a lot!

File: recipes/scikit-time/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - cython
+    - numpy
+    - scipy
```
Comment (beckermr) [inline recipes/scikit-time/meta.yaml]: Why do you need scipy in the host section? Is it used during the build of the package?

File: recipes/scikit-time/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+test:
+  imports:
+    - sktime 
+
+about:
+  home: https://github.com/scikit-time/scikit-time
+  license: LGPL 
+  license_file: LICENSE.txt
+  summary: 'Estimate and analyze Markov processes based on time series data. '
+
```
Comment (beckermr) [inline recipes/scikit-time/meta.yaml]: Remove this space.

File: recipes/scikit-time/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  #doc_url: https://notyet.there.to/
```
Comment (beckermr) [inline recipes/scikit-time/meta.yaml]: Remove this comment unless you want to keep it.

File: recipes/scikit-time/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    - pint
+    - scikit-learn
+
+test:
+  imports:
+    - sktime 
+
+about:
+  home: https://github.com/scikit-time/scikit-time
+  license: LGPL
```
Comment (beckermr) [inline recipes/scikit-time/meta.yaml]: Please use the proper SPDX identifier.

File: recipes/scikit-time/meta.yaml (hunk 1 above)
Comment (marscher) [reply recipes/scikit-time/meta.yaml]: exactly, we build some extension against the cython headers of blas provided by scipy.

File: recipes/scikit-time/meta.yaml (hunk 3 above)
Comment (marscher) [reply recipes/scikit-time/meta.yaml]: I'd like to keep it, so I won't forget to add the proper site once it is there.

File: recipes/scikit-time/meta.yaml (hunk 4 above)
Comment (marscher) [reply recipes/scikit-time/meta.yaml]: done

**Comment by marscher** [comment]
Thank you again!

**Pull request merged**